Dockerfile

```diff
@@ -1,18 +1,29 @@
-FROM docker.io/golang:1.23-bookworm as builder
+FROM docker.io/golang:1.23-bookworm AS builder
 
 ARG VER=devel
-ARG BUILDTAGS=""
 ARG EXTERNAL="0"
 
 ENV CGO_ENABLED=0
 
 RUN mkdir -p /sshpiperd/plugins
-WORKDIR /src
-RUN --mount=target=/src,type=bind,source=. --mount=type=cache,target=/root/.cache/go-build if [ "$EXTERNAL" = "1" ]; then cp sshpiperd /sshpiperd; else go build -o /sshpiperd -ldflags "-X main.mainver=$VER" ./cmd/... ; fi
-RUN --mount=target=/src,type=bind,source=. --mount=type=cache,target=/root/.cache/go-build if [ "$EXTERNAL" = "1" ]; then cp -r plugins /sshpiperd ; else go build -o /sshpiperd/plugins -tags "$BUILDTAGS" ./plugin/... ; fi
+WORKDIR /app
+RUN curl -fsSL https://go.dev/dl/go1.23.4.linux-amd64.tar.gz -o /tmp/go1.23.linux-amd64.tar.gz
+RUN tar -C /usr/local -xzf /tmp/go1.23.linux-amd64.tar.gz
```

Comment:
why don't we use the `1.23.4` go image specifically instead of downloading and setting up go?